src/box/lua/iproto.c

```diff
@@ -287,23 +166,32 @@ push_iproto_protocol_features(struct lua_State *L)
 	lua_pushinteger(L, IPROTO_CURRENT_VERSION);
 	lua_setfield(L, -2, "protocol_version");
 
-	const struct iproto_constant features[] = {
-		{"streams", IPROTO_FEATURE_STREAMS},
-        	{"transactions", IPROTO_FEATURE_TRANSACTIONS},
-        	{"error_extension", IPROTO_FEATURE_ERROR_EXTENSION},
-        	{"watchers", IPROTO_FEATURE_WATCHERS},
-        	{"pagination", IPROTO_FEATURE_PAGINATION},
-	};
+	size_t region_svp = region_used(&fiber()->gc);
+	size_t alloc_size;
+	struct iproto_constant *features = xregion_alloc_array(
+		&fiber()->gc, struct iproto_constant,
+		iproto_feature_id_constants_size, &alloc_size
+	);
+	static const char prefix[] = "IPROTO_FEATURE_";
```

Comment:
Oy vay... Please just use `const char *` or even inline (or define a macro) :joy:

Reply:
What's the problem? It allows me to calculate string length in compile time.

Reply:
Changed to strlen.